src/com/example/supabase.cljs

```diff
@@ -27,11 +25,14 @@
       .user
       (js->clj :keywordize-keys true)))
 
-;; .from('cities')
-;;   .insert([
-;;     { name: 'The Shire', country_id: 554 },
-;;     { name: 'Rohan', country_id: 555 },
-;;   ])
+(defn select-responses [client {:user/keys [id]}]
+  (go (try
+        (<p! (-> @client
+                 (.from "response_scores")
+                 (.select "*")
+                 (.eq "user_id" id)))
```

Comment:
pretty easy to select things and filter by user id.

This database has row level security so its makeing sure the currently logged in user is the same as `user_id`